sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -90,15 +92,14 @@ class DirectRunner(PipelineRunner):
   """Executes a single pipeline on the local machine."""
 
   # A list of PTransformOverride objects to be applied before running a pipeline
-  # using DirectRunner.
-  # Currently this only works for overrides where the input and output types do
-  # not change.
-  # For internal SDK use only. This should not be updated by Beam pipeline
-  # authors.
```

Comment:
<!--thread_id:cc_153932400_t; commit:022e74460433e0ad9b44356229151c0d7b0af351; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Why the change here?

<!--section:body-->
At the time of the 2.0 release we tried to normalize the message to "For internal use only; no backwards-compatibility guarantees." (https://issues.apache.org/jira/browse/BEAM-2241)